libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1935,6 +1936,12 @@ void GCS::send_textv(MAV_SEVERITY severity, const char *fmt, va_list arg_list, u
     if (spektrum != nullptr) {
         spektrum->queue_message(severity, first_piece_of_text);
     }
+#endif
+#if HAL_CRSF_TELEM_ENABLED
+    AP_CRSF_Telem* crsf = AP::crsf_telem();
+    if (crsf != nullptr) {
+        crsf->queue_message(severity, first_piece_of_text);
```

Comment:
Does queue_message cope with when custom telemetry is disabled?

Reply:
...very good point, we can skip pushing messages if not really needed